components/_headers.scss

```diff
@@ -32,7 +29,25 @@ h3 {
 		text-transform: uppercase;
 	}
 
+	&.size-up {
+		font-size: 1.25em;
+	}
+}
+
+h1, h2, h3, h4, h5, h6 {
 	&.center {
 		text-align: center;
 	}
+	&.lightweight {
```

Comment:
Line should be indented with spaces, not tabs<br>Each selector in a comma sequence should be on its own line